demos/src/simple-header.mustache

```diff
@@ -0,0 +1,60 @@
+<header class="o-header o-header--simple" data-o-component="o-header" data-o-header--no-js>
+    <div class="o-header__row o-header__top">
+        <div class="o-header__container">
+            <div class="o-header__top-wrapper">
+                <div class="o-header__top-column o-header__top-column--left">
+                        <a href="#menu" class="o-header__top-link o-header__top-link--menu" aria-controls="menu">
```

Comment:
Could you use the new menu bit from the new header.mustache that includes the drawer?